### DISCUSSION TITLE
GitLab MR Description via Mail

### DISCUSSION THREAD

**Discussion by fzoske** [renovatebot/renovate#9251] [Request Help] [answered]
Hi,

is it possible to configure renovate in a way, that the MR created in GitLab is directly assigned to one of assignee and not as an update to the MR?

Currently I just get the mail `Assignee changed to <user>` without any other content and it would be nice to have the description of the MR in the email as well. 

Best regards,
Fabian

**Comment by rarkins** [comment, marked as answer]
It's not currently possible, but please create it as a feature request issue so we can assess the requirements

**Reply by fzoske** [reply]
Thank you, done that.
